doc/build-osx.md

```diff
@@ -35,7 +38,7 @@ you can use [this](/contrib/install_db4.sh) script to install it
 like so:
 
 ```shell
-./contrib/install_db4.sh .
```

Comment:
The `.` is part of the command, it shouldn't be removed.

Reply:
It will be corrected in a future PR. I will try to articulate the issue better below.